### PR TITLE
Implement Hacker Login

### PR DESCRIPTION
**Pull request by omarchehab98**
https://hackthevalley.atlassian.net/browse/HV3HP-10

### CODE REVIEW

**Comment by omarchehab98** [comment]
```
File sizes after gzip:
  64.18 KB  build/static/js/main.0b96a5ed.js
  1.49 KB   build/static/css/main.9a241e66.css
```

**Comment by junzhengca** [comment]
Looks good, but I just updated the SDK, try using the SDK instead?

**Comment by omarchehab98** [comment]
```
File sizes after gzip:
  106.24 KB  build/static/js/main.b1c1dd4d.js
  1.49 KB    build/static/css/main.9a241e66.css
```

File: src/env.example.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "api": {
+    "protocol": "http",
+    "host": "127.0.0.1:3000"
+  }
+}
```
Comment (junzhengca) [inline src/env.example.json]: Is this still being used?

**Review by junzhengca — APPROVED** [review]
Looks good.

File: src/env.example.json (hunk 1 above)
Comment (omarchehab98) [reply src/env.example.json]: @junthehacker fixed in e0bfa96cb1bb8b04aefa9a19cb96cac89fd1ef35

**Pull request merged**